projects/mi_sensitive_human_data/MIROPA/schema/data_schema.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$id": "http://some-github-repo.url/schemas/data",
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "MIROPA schema for Data entity",
+    "description": " Sensitive human data as defined by GDPR",
+    "type": "object",
+    "properties": {
+        "@context": {
+            "description": "The JSON-LD context",
+            "anyOf": [
+                {
+                      "type": "string"
+                },
+                {
+                      "type": "object"
+                },
+                {
+                      "type": "array"
+                }
+                  ]
+          },
+          "@id": {
+            "description": "The JSON-LD identifier",
+            "type": "string",
+            "format": "uri" 
+          },
+          "@type": {
+            "description": "The JSON-LD type",
+            "type": "string",
+            "const": "Data"
+          },
+      
+      
+      "category_of_personal_data": {
+        "description": "  Any information which are related to an identified or identifiable natural person (Art. 4 (1) GDPR) .",
+        "type": "array",
+        "items" : {
+         "$ref": "person_schema.json#"
+        }
+      },
+      
+      "special _category_of_personal_data": {
+        "description": "Sensitive Personal Data whose use requires specific legal permission or justification such as genetic data, biometric data (where this is used for identification purposes), health data, etc.",
+       
+        "type": "array",
+        "items": {
+            "type": "string",
+            "enum": [
+              "Genetic",
+              "Biometric",
+              "Health",
+              "Sexual orientation",
+              "Convictions",
+              "Race",
+              "Other"
+            ]
+        
+      }
+    },
+      
+      "category_of_data_subjects": {
```

Comment:
Could you please go through the property names and use camelCase? Just for consistency with other schemas